### PR TITLE
[ZEPPELIN-3225] Minor fix for missing annotations

### PR DESCRIPTION
**Pull request by KirillTurchinskii**
### What is this PR for?
Some code refactoring is done, annotations added, and some of warnings has been removed

### What type of PR is it?
Refactoring

### Todos
Refactoring and Annotations added

###What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-3225?filter=12348928

### How should this be tested?
* https://travis-ci.org/github/KirillTurchinskii/zeppelin/builds/682564176

### Questions:
* Does the licenses files need update?
* Is there breaking changes for older versions?
* Does this needs documentation?

### CODE REVIEW

**Comment by alexott** [comment]
Can you please add Travis CI link?

**Comment by KirillTurchinskii** [comment]
Even master branch build is failed.
https://travis-ci.org/github/KirillTurchinskii/zeppelin/builds

вс, 3 мая 2020 г. в 11:38, Alex Ott <notifications@github.com>:

**Comment by alexott** [comment]
Can you restart only failed jobs? (click icon on the right of the job)

File: zeppelin-server/src/main/java/org/apache/zeppelin/realm/ZeppelinHubRealm.java [hunk 1]
```diff
@@ -173,7 +173,7 @@ protected User authenticateUser(String requestBody) {
    * @param pwd
    * @return
    */
-  protected String createLoginPayload(String login, char[] pwd) {
+  protected String createLoginPayload(String login, char ... pwd) {
```
Comment (Reamer) [inline zeppelin-server/src/main/java/org/apache/zeppelin/realm/ZeppelinHubRealm.java:176]: What is the reason for this change?

File: zeppelin-server/src/test/java/org/apache/zeppelin/cluster/ClusterEventTest.java [hunk 2]
```diff
@@ -66,9 +66,7 @@
 import java.util.Map;
 import java.util.Set;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertThat;
+import static org.junit.Assert.*;
```
Comment (Reamer) [inline zeppelin-server/src/test/java/org/apache/zeppelin/cluster/ClusterEventTest.java:69]: I would prefer a list of imports and no wildcard.

Comment (alexott) [reply zeppelin-server/src/test/java/org/apache/zeppelin/cluster/ClusterEventTest.java:69]: checkstyle plugin should catchup this, at least in main code....

**Comment by KirillTurchinskii** [comment]
New pull request 
https://github.com/apache/zeppelin/pull/3760
This has to many problems, so I decided to start over.
